### DISCUSSION TITLE
Filling auto-columns when importing data

### DISCUSSION THREAD

**Discussion by the-davidsn** [Budibase/budibase#11208] [Ideas]
Hi everyone, 

first: i had a great experience with Budibase so far. :) 

I am currently working on a PoC to provide a simple UI for our internal business user to provide some Masterdata for business entities. Our approach is to replace all the messy excel files with very simple CRUD-Applications + Automations. There are several reasons going that way - one of them is the audit which is given through the auto-colums. Unfortunately, not all of these columns will be filled if we start with an empty table (currently using Budibase DB) and importing a CSV file. It would be awesome having all columns filled - or at least the option doing that within the import process. Currently the `Created By` and `Updated At` won't be filled. 

Assume a user imports a CSV file to a table then the column `Created By` will filled with the current User relationship and the column `Updated By` will be filled with the current User relationship to have an holistic audit log. 

Thanks in advance for spending effort to understand my idea. If you have any further questions i would be happy to discuss them. :) 

EDIT: `Created At` and `Updated At` will be filled automatically, therefore just the `Created By` and `Updated By` need to be filled as well. Adjusted the idea :)

**Comment by ConorWebb96** [comment]
Hey @DavidHuegel,

I don't believe this is something which should be done as you wouldn't always want to assign the createdBy and updatedBy to the user whom imports the CSV.

Now if this were to be done, I'd likely recommend adding a toggle within the CSV importing modal `Use importer as creator` (the text for this would need to be changed likely). Whenever this toggle is checked it would then use the person who is importing the CSV as the link for the createBy and updatedBy fields.

With this all being said, I'd recommend creating a feature request if this is something which you would want and that others are likely to want. You can do this by going here and click the feature request option. https://github.com/Budibase/budibase/issues/new/choose

I hope this helps!
